tests/pathfinding/fixtures/__init__.py

```diff
@@ -12,3 +13,8 @@
 @pytest.fixture
 def token_network_model() -> TokenNetwork:
     return TokenNetwork(TokenNetworkAddress(bytes([1] * 20)))
+
+
+@pytest.fixture
+def address_to_reachability() -> Dict[Address, AddressReachability]:
+    return {}
```

Comment:
Does this fixture really help? Just putting `{}` inside the test would both shorter and one indirection less.

Reply:
The thing is that it gets populated by the `populate_token_network_case_X` fixtures, so I think it's necessary (but we should improve that setup system)